Rock/Security/Authorization.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// <copyright>
-// Copyright by the Spark Development Network
+// Copyright 2013 by the Spark Development Network
```

Comment:
We recently removed the year from all of the copyright statements, looks like it somehow came back here.